CODING_STANDARDS.md

```diff
@@ -244,12 +244,83 @@ activateRipple() {
 }
 ```
 
+##### Selectors
+* Component selectors should be lowercase and delimited by hyphens.
+* Directive selectors should be camel cased.
```

Comment:
I would add "Exceptions may be made for directives that _act_ like a component but would have an empty template, or when the directive is intended to match some existing attribute."